app/src/main/java/com/nutomic/syncthingandroid/activities/DeviceActivity.java

```diff
@@ -167,6 +180,7 @@ public void onCreate(Bundle savedInstanceState) {
         mCompressionContainer = findViewById(R.id.compressionContainer);
         mCompressionValueView = findViewById(R.id.compressionValue);
         mIntroducerView = findViewById(R.id.introducer);
+        mDevicePaused = findViewById(R.id.devicePauseSync);
```

Comment:
Just devicePause, there is only one type of pause, so adding the word sync just makes it confusing.

Reply:
Ok.